contracts/Bdx/BDXShares.sol

```diff
@@ -37,21 +37,17 @@ contract BDXShares is ERC20Upgradeable, OwnableUpgradeable {
         }
     }
 
-    function mint(
-        address bd_stable,
-        address to,
-        uint256 amount
-    ) public onlyOwner {
+    function mint(address to, uint256 amount) public onlyOwner {
```

Comment:
@AndrzejGisWideLab can you think of any good reason this was here before? We just sent to it the zero address...

Reply:
I think originally the mint function was supposed to only called for a bdStable and it hasn't been cleaned up.